### PR TITLE
Add ControlNet and SDXL pipelines to demoDiffusion

### PR DESCRIPTION
**Pull request by rajeevsrao**

### CODE REVIEW

**Comment by artisan323** [comment]
I've been testing this code and the performance does not meet my expectations in terms of speed. Could you please take a look and see if there's any issue? Thanks.

> python3 demo_txt2img_xl.py "(Anime), A cute little boy standing bent and bowing" --negative-prompt "badly drawn face:1.8,face sharpen:1.3,low quality:1.5,worst quality, badly drawn hand, bad anatomy, bad proportions" --hf-token=$HF_TOKEN --version=xl-1.0 --denoising-steps 10 --use-cuda-graph --build-static-batch --num-warmup-runs 5 --nvtx-profile
[I] Initializing TensorRT accelerated StableDiffusionXL txt2img pipeline
[I] Load tokenizer pytorch model from: pytorch_model/xl-1.0/SD_XL_BASE/tokenizer_2
[I] Load tokenizer pytorch model from: pytorch_model/xl-1.0/SD_XL_BASE/tokenizer
Loading TensorRT engine: engine_xl_base/clip.plan
[I] Loading bytes from engine_xl_base/clip.plan
Loading TensorRT engine: engine_xl_base/clip2.plan
[I] Loading bytes from engine_xl_base/clip2.plan
Loading TensorRT engine: engine_xl_base/unetxl.plan
[I] Loading bytes from engine_xl_base/unetxl.plan
Loading TensorRT engine: engine_xl_base/vae.plan
[I] Loading bytes from engine_xl_base/vae.plan
[W] CUDA lazy loading is not enabled. Enabling it can significantly reduce device memory usage and speed up TensorRT initialization. See "Lazy Loading" section of CUDA documentation https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#lazy-loading
[I] Warming up ..
[I] Running StableDiffusion pipeline
SD-XL Base Pipeline
|------------|--------------|
|   Module   |   Latency    |
|------------|--------------|
|    CLIP    |      9.75 ms |
| UNet x 10  |   2689.36 ms |
|  VAE-Dec   |    295.88 ms |
|------------|--------------|
|  Pipeline  |   2998.44 ms |
|------------|--------------|
Throughput: 0.33 image/s
return_type:image
Saving image 1 / 1 to: output/txt2img-xl-fp16-(Anime),_A-1-5692.png
time cost3.0430641174316406
|------------|--------------|
|    e2e     |   2998.44 ms |
|------------|--------------|


## Environment



**TensorRT Version**: 8.6.1

**NVIDIA GPU**: A10

**NVIDIA Driver Version**: 520.61.05

**CUDA Version**: 11.8

**CUDNN Version**: 8.9.3.28

**Comment by yjinhui** [comment]
Is your environment written incorrectly？
I try it as same as your readme, but image:nvcr.io/nvidia/pytorch:23.06-py3 is incompatible to NVIDIA Driver Version: 520.61.05.

**Comment by artisan323** [comment]
I am not using this image because I cannot install cuda12 due to environmental issues. Therefore, I am using the TRT 8.6.1 version of cuda11.8, and the image is created by myself.

File: demo/Diffusion/README.md [hunk 1]
```diff
@@ -76,33 +76,63 @@ To download the model checkpoints for the Stable Diffusion pipeline, you will ne
...
+
+Run the below command to generate an image with Stable Diffusion XL
+
+```bash
+python3 demo_txt2img_xl.py "a beautiful photograph of Mt. Fuji during cherry blossom" --hf-token=$HF_TOKEN --version=xl-1.0
+```
+
+
+### Universal input arguments
 - One can set schdeuler using `--scheduler=EulerA`. Note that some schedulers are not available for some pipelines or version.
```
Comment (msustik) [inline demo/Diffusion/README.md:132]: typo change to scheduler

**Pull request closed without merging**